candle-examples/examples/vgg16/README.md

```diff
@@ -0,0 +1,11 @@
+
+
+```bash
+python3 export_models.py
```

Comment:
We would rather get all the examples to be self-contained through the use of the `hf-hub` crate, see the other examples. In this case, you can find some sample model weights [here](https://huggingface.co/timm/vgg16.tv_in1k/tree/main).